vendor/modules.txt

```diff
@@ -1162,7 +1161,7 @@ gopkg.in/yaml.v2
 # gopkg.in/yaml.v3 v3.0.1
 ## explicit
 gopkg.in/yaml.v3
-# k8s.io/api v0.28.2 => k8s.io/api v0.28.2
+# k8s.io/api v0.30.1 => k8s.io/api v0.28.2
```

Comment:
Down here go does the right thing and downgrades k8s.

We should upgrade k8s in a follow up PR for OCP 1.17.